packages/base/src/signals/signal-manager.ts

```diff
@@ -143,6 +144,9 @@ export class SignalManager extends EventEmitter implements SignalManager {
     fireRedoSignal(): void {
         this.emit(Signals.REDO);
     }
+    fireOutputDataChanged(): void {
+        this.emit(Signals.OUTPUT_DATA_CHANGED);
```

Comment:
Should the signal have a payload to indicate which data provider or data providers were changed? This would allow receivers to filter out signals and only act on it if it is meant for them? 

WDYT?

Reply:
Yeah that is a good idea. I will add that in the next patch.